profile/2017-03-09-profile.json

```diff
@@ -0,0 +1,47 @@
+{
+    "resource-manager": {
+        "microsoft.authorization": {
+            "version": "2015-07-01",
+            "types": {
+                "locks":"2015-01-01",
+                "policyassignments": "2015-10-01-preview",
+                "policydefinitions": "2015-10-01-preview"
+            }
+        },
+        "microsoft.compute": {
+            "version": "2016-03-30",
+            "types": {}
```

Comment:
We must have `virtualMachineScaleSets` here with 2015-06-15 (the Network ApiVersion). This file should not replicate the Swagger mistake of considering that `virtualMachineScaleSets` is a Network RP.